src/ui/public/agg_types/buckets/histogram.js

```diff
@@ -47,7 +47,7 @@ define(function (require) {
           write: function (aggConfig, output) {
             var val = aggConfig.params.extended_bounds;
 
-            if (val.min != null || val.max != null) {
+            if (aggConfig.params.min_doc_count) {
```

Comment:
In #5960 I keep in the min and max check. The `min_doc_count` is used to check whether the checkbox has been selected. It seems the default is 1, and checking the box sets it to 0, which makes sense, since the extended bounds don't require documents.